content/blog/2025-03-18-parquet-pruning.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title: Parquet pruning in DataFusion: Read Only What Matters
+date: 2025-03-18
+author: Xiangpeng Hao
+categories: [performance]
+---
+
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+
+_Editor's Note: This blog was first published on [Xiangpeng Hao's blog]. Thanks to [InfluxData] for sponsoring this work as part of his PhD funding._
+
+[Xiangpeng Hao's blog]: https://blog.xiangpeng.systems/posts/parquet-to-arrow/
+[InfluxData]: https://www.influxdata.com/
+<hr/>
+
+Parquet has become the industry standard for storing columnar data, and reading Parquet efficiently is crucial for query performance.
```

Comment:
I recommend adding links and referring to this as Apache Parquet and Apache DataFusion on first mention

```suggestion
[Apache Parquet] has become the industry standard for storing columnar data, and reading Parquet efficiently is crucial for query performance.

[Apache Parquet]: https://parquet.apache.org/
```